docs/guide/pagination.md

```diff
@@ -12,20 +12,190 @@ An interface for limit/offset pagination can be use for basic pagination needs:
 @strawberry_django.type(models.Fruit, pagination=True)
 class Fruit:
     name: auto
+
+
+@strawberry.type
+class Query:
+    fruits: list[Fruit] = strawberry_django.field()
+```
+
+Would produce the following schema:
+
+```graphql title="schema.graphql"
+type Fruit {
+  name: String!
+}
+
+type Query {
+  fruits(pagination: PaginationInput): [Fruit!]!
+}
 ```
 
+And can be queried like:
+
 ```graphql title="schema.graphql"
 query {
   fruits(pagination: { offset: 0, limit: 2 }) {
     name
-    color
   }
 }
 ```
 
-There is not default limit defined. All elements are returned if no pagination limit is defined.
+The `pagination` argument can be given to the type, which will enforce the pagination
+argument every time the field is annotated as a list, but you can also give it directly
+to the field for more control, like:
+
+```python title="types.py"
+@strawberry_django.type(models.Fruit)
+class Fruit:
+    name: auto
+
+
+@strawberry.type
+class Query:
+    fruits: list[Fruit] = strawberry_django.field(pagination=True)
+```
+
+Which will produce the exact same schema.
+
+> [!NOTE]
+> There is no default limit defined. All elements are returned if no pagination limit is defined.
```

Comment:
should we enforce a default limit? (I'd say yes :D)

Reply:
I think we should, yes!

The idea in this PR was to add the `Paginated` only, without breaking the existing pagination we currently have, which would be a breaking change.

Do you think we should also do that here or in a following PR?

Reply:
Done! :)